auth/niauth.yaml

```diff
@@ -299,6 +299,10 @@ paths:
           name: type
           description: Filters the policy-templates by type. Can be a comma-separated list of types.
           type: string
+        - in: query
```

Comment:
we should probably add this parameter to our policies API as well to be consistent.

Reply:
done!